spring-ai/postman/spring-ai.postman_collection.json

```diff
@@ -0,0 +1,56 @@
+{
+	"info": {
+		"_postman_id": "f4282fac-bfe5-45b9-aae6-5ea7c43528ee",
+		"name": "spring-ai",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json",
+		"_exporter_id": "9576856"
+	},
+	"item": [
+		{
+			"name": "Generate poetry with genre and theme",
```

Comment:
Do we need all these spaces? maybe better identation?

Reply:
It's generated, so as for me it's should be like that

Reply:
its fine, tks